src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -183,15 +178,15 @@ void courthousesPut() throws Exception {
             .contentType(MediaType.APPLICATION_JSON_VALUE);
 
         mockMvc.perform(requestBuilder).andExpect(status().is2xxSuccessful())
-            .andExpect(jsonPath("$.courthouse_name", is("SWANSEA")))
-            .andExpect(jsonPath("$.code", is(457)))
+            .andExpect(jsonPath("$.courthouse_name", is("test")))
```

Comment:
what was the reason for changing the test values? I think its a good idea for tests to use real data where we can.

Reply:
the test is testing the put values. the SWANSEA value was already in the database, and so couldn't be used as it causes a unique constraint violation. It accepts any string, and will not exist after this test, so doesn't matter what string is used...as long as it isn't swansea. i need to make sure that its a me that isn't going to be added to flyway in the future, as then the test would break.